src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,49 @@
+@import "../../../../public/variables";
+
+@mixin backround($color,$shape) {
+  background-color: #{$color};
+  clip-path: $shape;
+  padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
+  border-radius: $euiSizeXS - 2;
+}
+
+.osdHeaderBreadcrumbs {
+  [data-background="osdHeaderBreadcrumbs first"] {
+    @include backround($osdSanFranciscoFog, polygon(0% 0%,100% 0%,calc(100% - #{
+      $euiSizeS + 2}) 100%,0% 100%));
+  }
+
+  [data-background="osdHeaderBreadcrumbs middle"] {
+    @include backround($osdSanFranciscoFog,
+    polygon($euiSizeS + 2 0%,100% 0%,calc(100% - #{$euiSizeS + 2})  100%,0% 100%));
```

Comment:
nit: Surprised this passed the sass inter but the spacing seems inconsistent.

Reply:
let me fix that